Source/WebCore/animation/CSSPropertyAnimation.cpp

```diff
@@ -4025,40 +4025,30 @@ static Ref<CSSCustomPropertyValue> blendedCSSCustomPropertyValue(const RenderSty
     return CSSCustomPropertyValue::create(blendingContext.progress < 0.5 ? from : to);
 }
 
-static std::pair<const CSSCustomPropertyValue*, const CSSCustomPropertyValue*> customPropertyValuesForBlending(const Document* document, const AtomString& customProperty, const CSSCustomPropertyValue* fromValue, const CSSCustomPropertyValue* toValue)
+static std::pair<const CSSCustomPropertyValue*, const CSSCustomPropertyValue*> customPropertyValuesForBlending(const Document* document, const AtomString& customProperty, const RenderStyle& fromStyle, const RenderStyle& toStyle)
 {
-    // FIXME: it would be convenient if RenderStyle stored the initialValue
-    // for a custom property that was not explicitly set.
-    auto initialValue = [&]() -> const CSSCustomPropertyValue* {
-        if (document) {
-            if (auto registered = document->customPropertyRegistry().get(customProperty))
-                return registered->initialValue();
-        }
-        return nullptr;
-    };
+    // FIXME: When would there be no document?
```

Comment:
You could add an `ASSERT()` and find out from tests :)

Reply:
I was more of hoping that document would be a non-nullable type.